src/AppBundle/Repository/ActivityRepository.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace AppBundle\Repository;
+
+/**
+ * ActivityRepository
+ *
+ * This class was generated by the Doctrine ORM. Add your own custom
+ * repository methods below.
+ */
+class ActivityRepository extends \Doctrine\ORM\EntityRepository
```

Comment:
'extends EntityRepository' would look better here. Use 'use' before the class :)